geomesa-accumulo/geomesa-accumulo-dist/pom.xml

```diff
@@ -155,7 +167,7 @@
                         <phase>package</phase>
                         <configuration>
                             <attach>true</attach>
-                            <finalName>geomesa-accumulo-${project.version}</finalName>
+                            <finalName>geomesa-accumulo_${scala.binary.version}-${project.version}</finalName>
```

Comment:
I just confined hardcoding to `project.artifactId` and `parent.project.artifactId`.  I'd prefer not to hard code but it's required for now